source/docs/publish/action.md

```diff
@@ -40,7 +40,7 @@ jobs:
   validate-hacs:
     runs-on: "ubuntu-latest"
     steps:
-      - uses: "actions/checkout@v3"
+      - uses: "actions/checkout@v4"
```

Comment:
```suggestion
```
Checking out the repository is not needed.
<https://github.com/hacs/action#example>